index.html

```diff
@@ -209,7 +207,7 @@ <h4>Dubbing <a>scripts</a></h4>
           <p>The <dfn>dubbing</dfn> process which consists in creating a <a>dubbing script</a>
             is a complex, multi-step process involving:</p>
           <ul>
-            <li>Transcribing and timing the dialogue in the original language from a completed programme to create a <a>transcript</a>;</li>
+            <li>Transcribing and timing the dialogue in its own language from a completed programme to create a <a>transcript</a>;</li>
```

Comment:
I changed it because "original language" is a defined term, and it would be confusing to use it here in a non-defined term sense.

Reply:
It seems to me that it would be precisely in the defined term sense here. What confused me is the term 'own'. After reading it again, I'm fine with it.